pkg/apis/pipeline/v1alpha1/artifact_bucket.go

```diff
@@ -116,7 +117,7 @@ func (b *ArtifactBucket) GetSecretsVolumes() []corev1.Volume {
 	volumes := []corev1.Volume{}
 	for _, sec := range b.Secrets {
 		volumes = append(volumes, corev1.Volume{
-			Name: fmt.Sprintf("bucket-secret-volume-%s", sec.SecretName),
+			Name: names.SimpleNameGenerator.GenerateName(fmt.Sprintf("bucket-secret-%s", sec.SecretName)),
```

Comment:
i had one thought, maybe overkill tho: we could make a `GenerateNamef`, something like:

```
names.SimpleNameGenerator.GenerateNamef("bucket-secret-%s", sec.SecretName)
```

Since we're so often passing the output of `fmt.Sprintf` in here anyway :woman_shrugging:

Reply:
nice idea